examples/Echo.Server/EchoServerSettings.cs

```diff
@@ -8,11 +8,7 @@ public static class EchoServerSettings
     {
         public static bool IsSsl
         {
-            get
-            {
-                string ssl = ConfigurationManager.AppSettings["ssl"];
-                return !string.IsNullOrEmpty(ssl) && bool.Parse(ssl);
-            }
+            get { return ConfigurationManager.AppSettings["ssl"].ToLower() == "true"; }
```

Comment:
ConfigurationManager.AppSettings doesn't return empty string when Object doesn't Exists?
(Idk if return i didn't checked at all) Using the Null Conditional Operator will be that avoided?